extension/scripts/features/collapsibleareas/ttCollapsibleAreas.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Collapse Areas",
+		"sidebar",
+		() => settings.pages.sidebar.collapseAreas,
+		null,
+		addCollapseIcon,
+		removeCollapseIcon,
+		{
+			storage: ["settings.pages.sidebar.collapseAreas"],
+		},
+		null
+	);
+
+	async function addCollapseIcon() {
+		await requireSidebar();
+		if (!document.find("[class='header-arrow']") && !mobile) {
+			let areasIElement = document.newElement({ type: "i", class: "icon fas fa-caret-down" });
```

Comment:
If you aren't reassigning something, you should use `const` for it in v6. It's an extra safety, so you know that it won't be edited later on.

Reply:
This applies on everything, not just this part of the code.